docs/how-to-restart-server.md

```diff
@@ -1 +1,12 @@
-# How do I restart my stardew server?
+# How do I restart my Junimohost Stardew server?
+
+We understand that occasional bugs can hinder your gameplay, but fret not – we've got a quick and easy solution for you. The best place to start when troubleshooting issues with your Junimohost server is to restart the server. This can be accomplished by following the below steps.
+
+What are the steps? (PC)
```

Comment:
(PC) is redundant info, they're visiting a website. this entire line is redundant, you can just list the steps.